### DISCUSSION TITLE
Where is data stored?

### DISCUSSION THREAD

**Discussion by p-zach** [bmeares/Meerschaum#58] [Q&A] [answered]
I was poking around the file system trying to see where the data is stored for my plugins, but I couldn't find it. In case I just want to take a look, where can I access the files that hold my plugins' data?

**Comment by bmeares** [comment, marked as answer]
Hi @p-zach, do you mean the pipes' data or the configuration files?

The pipes are stored in the database of your instance (e.g. `sql:main` is the database to the default [Meerschaum stack](https://meerschaum.io/reference/stack/)).

The config files are stored in `~/.config/meerschaum/config/`, and you can open them with `mrsm edit config` (e.g. `mrsm edit config plugins` to open the plugins YAML file).
